components/patient-data-sharing/push-api/src/main/resources/ApplicationResources_ar.properties

```diff
@@ -1,8 +1,4 @@
 #X-Generator: crowdin.com
-phenotips.patientSharing.push.dialog.serverSelection.label.multiplePatients=\u0625\u0644\u0649 \u0623\u064a\u0646 \u064a\u062a\u0645 \u0625\u0631\u0633\u0627\u0644 \u062c\u0645\u064a\u0639 \u0628\u064a\u0627\u0646\u0627\u062a \u0627\u0644\u0645\u0631\u064a\u0636\u061f
-phenotips.patientSharing.push.dialog.serverSelection.hint.multiplePatients=\u062e\u0627\u062f\u0645 \u0641\u064a\u0646\u0648\u062a\u064a\u0628\u0633 \u0627\u0644\u0628\u0639\u064a\u062f \u062d\u064a\u062b \u0633\u064a\u062a\u0645 \u0625\u0631\u0633\u0627\u0644 \u0643\u0627\u0641\u0629 \u0627\u0644\u0628\u064a\u0627\u0646\u0627\u062a.
-phenotips.patientSharing.push.dialog.serverSelection.label.singlePatient=\u0625\u0644\u0649 \u0623\u064a\u0646 \u064a\u062a\u0645 \u0625\u0631\u0633\u0627\u0644 \u0645\u0644\u0641 \u062a\u0639\u0631\u064a\u0641 \u0647\u0630\u0627 \u0627\u0644\u0645\u0631\u064a\u0636\u061f
-phenotips.patientSharing.push.dialog.serverSelection.hint.singlePatient=\u062e\u0627\u062f\u0645 \u0641\u064a\u0646\u0648\u062a\u064a\u0628\u0633 \u0627\u0644\u0628\u0639\u064a\u062f \u062d\u064a\u062b \u0633\u064a\u062a\u0645 \u0625\u0631\u0633\u0627\u0644 \u0628\u064a\u0627\u0646\u0627\u062a \u0647\u0630\u0627 \u0627\u0644\u0645\u0631\u064a\u0636.
```

Comment:
Just FYI, you don't have to delete deprecated translations from other languages anymore. Our Crowdin integration removes them automatically and syncs all the keys with the default language.